test/NuGet.Clients.Tests/NuGet.CommandLine.Test/PortReserver.cs

```diff
@@ -22,6 +23,7 @@ namespace NuGet.CommandLine.Test
     public class PortReserver : IDisposable
```

Comment:
Can we just drop-in replace this with:
https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Test.Server/PortReserver.cs